irohad/torii/processor/impl/transaction_processor_impl.cpp

```diff
@@ -95,16 +95,15 @@ namespace iroha {
     }
 
     void TransactionProcessorImpl::transactionHandle(
-        std::shared_ptr<model::Transaction> transaction) {
+        std::shared_ptr<shared_model::interface::Transaction> transaction) {
       log_->info("handle transaction");
       model::TransactionResponse response;
-      response.tx_hash = hash(*transaction).to_string();
+      response.tx_hash =
```

Comment:
`STATELESS_VALIDATION_SUCCESS` is set in caller method of `CommandService`, so it's possible to remove response creation and `on_next` call here.